dashboard/sample_metadata_classifier.py

```diff
@@ -231,5 +231,14 @@ def interpret(project, papers, bits):
             #  admission.
             date="2020-01",
             collection="bronchoalveolar lavage fluid")
+    elif project in papers["Bohl 2022"]["projects"]:
+        sample, = bits
+        return sample, dict(
+                country="Cambodia",
+                city="Chbar Mon",
+                state="Kampong Speu",
+                # Patient samples collected between March 2019 and October 2020
+                date="2020",
+                collection="sera")
```

Comment:
Would "serum", "blood", or "blood serum" be clearer?

Reply:
Changed to "blood serum" in [3188f70](https://github.com/naobservatory/mgs-pipeline/pull/14/commits/3188f70679fc4dda744582fa7ce7032b4a07d9cd)